src/filters/SelectSorting/getSortingText.ts

```diff
@@ -0,0 +1,53 @@
+import { FilterState } from "../reducer";
+
+/**The list of sort by options and their corresponding labels. */
+export const sortBy = {
+  name: "Committee Name",
+  date: "Event Date",
+  value: "Search Relevance",
+} as const;
```

Comment:
Oh my other comment about storybook files being parameters passed into the component seems to be incorrect given this. I think these values may need to be removed / changed given UX feedback.

Reply:
I believe the UX team wanted sort by `Most recent` or `Most relevant`  only for the search page.
I'm not sure about what sorting is wanted for the all events page.

There is definitely no Ascending or Descending options for either page though.

Reply:
https://projects.invisionapp.com/share/UGW7LLV87VQ#/screens/407317059
I'm assuming when the user clicks on `Most relevant`, `Most recent` and `Most relevant` checkboxes will appear.
https://projects.invisionapp.com/share/UGW7LLV87VQ#/screens/411371408

There aren't any sorting options on the all events page though.